test/unit/models/users.test.js

```diff
@@ -115,6 +115,16 @@ describe("users", function () {
       const { user } = await users.fetchUser({ githubUsername });
       expect(user).to.haveOwnProperty("github_created_at");
     });
+
+    it("it should filter out the id field while updating profileDiff", async function () {
+      const userData = userDataArray[0];
+      const { userId } = await users.addOrUpdate(userData);
```

Comment:
we should write test cases with an invalid or missing message field in the request body.